package.json

```diff
@@ -47,6 +48,15 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "rebuild": "node-gyp rebuild",
     "prebuild": "prebuildify -t 8.14.0 -t electron@3.0.0 --napi --strip",
+    "prebuild:cross:android-arm": "prebuildify-cross --target android-arm",
+    "prebuild:cross:android-arm64": "prebuildify-cross --target android-arm64",
+    "prebuild:cross:linux-armv6": "prebuildify-cross --target linux-armv6",
+    "prebuild:cross:linux-armv7": "prebuildify-cross --target linux-armv7",
+    "prebuild:cross:linux-arm64": "prebuildify-cross --target linux-arm64",
+    "prebuild:cross:linux-x64": "prebuildify-cross --target linux-x64",
+    "prebuild:cross:linux-x86": "prebuildify-cross --target linux-x86",
+    "prebuild:cross:windows-x64": "prebuildify-cross --target windows-x64",
+    "prebuild:cross:windows-x86": "prebuildify-cross --target windows-x86",
```

Comment:
@ahdinosaur You can remove all `*-x86` targets, we've decided to not support 32 bits in `leveldown`.